File: 07.08/teste/app/src/main/java/com/example/daniel/teste/MainActivity.java
package com.example.daniel.teste;

import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.util.Log;
import android.view.View;
import android.widget.EditText;
import android.widget.Toast;

import com.parse.FindCallback;
import com.parse.ParseException;
import com.parse.ParseObject;
import com.parse.ParseQuery;
import com.parse.SaveCallback;

import org.json.JSONArray;
import org.json.JSONException;
import org.json.JSONObject;
import org.json.JSONStringer;

import java.util.List;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }

    public void logIn(View view) {
        EditText texto = findViewById(R.id.editText);

        final JSONArray ja = new JSONArray();
        ja.put(1);
        ja.put(2);
        ja.put(3);
        ja.put(5);

        final String[] parseId = new String[1];
        ParseQuery<ParseObject> query = ParseQuery.getQuery("UserActivity");
        query.findInBackground(new FindCallback<ParseObject>() {
            @Override
            public void done(List<ParseObject> objects, ParseException e) {
                if(e == null) {
                    if(objects.size() > 0) {
                        int i = 1;
                        for(ParseObject object : objects) {
                            object.getString("level");
                            object.put("level", ja);
                            object.saveInBackground(new SaveCallback() {
                                @Override
                                public void done(ParseException e) {
                                    Toast.makeText(MainActivity.this, "Just beautiful.", Toast.LENGTH_SHORT).show();
                                }
                            });
                        }
                    } else {
                        Toast.makeText(MainActivity.this, "Não existe", Toast.LENGTH_SHORT).show();
                    }
                } else {
                    Toast.makeText(MainActivity.this, e.toString(), Toast.LENGTH_SHORT).show();
                }
            }
        });
    }
}

File: kotlin/HelloWorld/app/src/main/java/com/example/helloworld/MainActivity.kt
package com.example.helloworld

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.Button
import android.widget.TextView
import android.widget.Toast
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        //val btn = findViewById<Button>(R.id.button)
        //val txt = findViewById<TextView>(R.id.textView);
        var count = 0;

        button.setOnClickListener {
            count++
            textView.text = count.toString()
           // Toast.makeText(this@MainActivity, "what", Toast.LENGTH_SHORT)
        }
    }
}